File: Curso_POO/PHP/Car.php
<?php
require_once("Account.php");
class Car {
public $id=integer;
public $license=string;
public $driver=string;
protected $passenger=integer;

public function __construct($license,$driver){
    $this->license=$license;
    $this->driver=$driver;
}
public function printDataCar(){
    echo "license: $this->license,
    conductor:.{$this->driver->name},
    document:{$this->driver->document}";
}

public function getPassenger() {
    return $this->passenger;
}

public function setPassenger($passenger) {
        
    if ($passenger == 4) {
        $this->passenger = $passenger;
    }
    else {
        echo "Necesitas asignar 4 pasajeros";
    }

}
}

File: Curso_POO/Java/Car.java
class Car {
    private Integer id;
    public String license;
    public Account driver;
    protected Integer passenger;
    
    public Car(String license,Account driver){
        this.license=license;
        this.driver=driver;
    
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getPassenger(){
        return passenger;
    }
    
    public void setPassenger(Integer passenger){
        if(passenger==4){
         this.passenger=passenger;
        }else{
         System.out.println("Necesitas asignar 6 pasajeros");
        }  
    }
    

}

